Heroku/package.json

```diff
@@ -34,6 +34,6 @@
 		"devTest:backend": "gradlew :backend:endpoint:run --args=\"-P:twisterrob.cinema.staticRootFolder=../../deploy/frontend/static -P:twisterrob.cinema.fakeRootFolder=../../config/fake-data\""
 	},
 	"devDependencies": {
-		"npm-run-all": "4.1.5"
+		"npm-run-all2": "^6.1.1"
```

Comment:
Investigate how replace works in Renovate. Use it on okhttp, and stdlib-jre

Reply:
https://github.com/renovatebot/renovate/blob/39d01251b26bd6f84c34d45e834249418f6f8210/lib/config/presets/internal/replacements.ts#L621

Reply:
https://github.com/TWiStErRob/renovate-config/issues/21